ckcp/openshift_dev_setup.sh

```diff
@@ -332,20 +348,25 @@ patches:
     --kcp-org "$kcp_org" \
     --kcp-workspace "$kcp_workspace" \
     --work-dir "$WORK_DIR" \
-    --kustomization "$GIT_URL/gitops/kcp/pac-manager?ref=$GIT_REF" |
+    --kustomization "$GIT_URL/gitops/kcp/pac-manager?ref=$GIT_REF" 2>&1|
```

Comment:
With the latest changes, my understanding is that this change can be reverted.

Reply:
Reverted now.